apps/hvn/app/[locale]/application/page.tsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react';
+import { Button } from '@fc/ui/base/button';
+import { Input } from '@fc/ui/base/input';
+import { useTranslations } from 'next-intl';
+
+const ApplicationPage = () => {
+  const t = useTranslations('Application');
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [acceptedTerms, setAcceptedTerms] = useState(false);
+
+  const cities = ['Amsterdam', 'Den Haag', 'Noord Brabant'];
+  const events = ['Harmonie van Nederland - Amsterdam'];
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    if (!acceptedTerms) {
+      alert('You must accept the terms of use to proceed.');
+      return;
+    }
+    setIsSubmitting(true);
+    // Implement form submission logic here
+    console.log('Application submitted');
+    setIsSubmitting(false);
+  };
+
+  return (
+    <section className="min-h-screen pt-20 pb-10 bg-gray-50">
+      <div className="container mx-auto px-4">
+        <div className="text-center mb-16">
+          <h1 className="text-4xl font-bold mb-4">{t('title')}</h1>
+          <p className="text-muted-foreground max-w-2xl mx-auto">{t('subtitle')}</p>
+        </div>
+
+        <div className="bg-white rounded-lg shadow-lg p-8 space-y-6">
```

Comment:
Avoid using bg-white, text-black etc. To support dark theme switch, we have `text-foreground`, `bg-background`, `text-primary`, `text-primary-foreground`, `text-muted`, `text-muted-foreground` etc.